src/main/java/net/gegy1000/spleef/game/map/SpleefMapConfig.java

```diff
@@ -1,18 +1,25 @@
 package net.gegy1000.spleef.game.map;
 
+import java.util.HashMap;
+import java.util.Map;
 import com.mojang.serialization.Codec;
 import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.gegy1000.spleef.game.map.shape.CircleShape;
+import net.gegy1000.spleef.game.map.shape.MapShape;
+import net.gegy1000.spleef.game.map.shape.SquareShape;
 import net.minecraft.block.BlockState;
 import net.minecraft.block.Blocks;
 
 public final class SpleefMapConfig {
+    private static final Map<String, Class<? extends MapShape>> SHAPE_TYPES = new HashMap<>();
```

Comment:
not totally necessary, but can use `TinyRegistry` here (which is essentially just a `HashMap` that implements `Codec`)

Reply:
Fixed in 7fe297227f385c81c9645d230d405a2fe3a2cc24